Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Elements/ElementFactoryTests.cs

```diff
@@ -37,9 +37,6 @@ public void Should_FindElements_ViaElementFactory()
             Assert.IsTrue(Factory.FindElements<Button>(By.XPath("//*")).Count > 1);
         }
 
-        public static IButton GetButton(IElementFactory coreFactory, By loc, string nam)
-        {
-            return coreFactory.GetButton(loc, nam);
-        }
+        public static IButton GetButton(IElementFactory coreFactory, By loc, string nam) => coreFactory.GetButton(loc, nam);
```

Comment:
As I can see you prefer to use expression body for methods.
It's okay here, but be sure to check input parameters when it's required

Reply:
will do